include/mq/api/MacroAPI.h

```diff
@@ -140,6 +182,19 @@ inline namespace deprecated
 #pragma warning(suppress: 4191)
 		return AddMQ2Data(szName, (fMQData)Function);
 	}
+
+	//DEPRECATE("ParseMacroParameter no longer takes a PlayerClient parameter. It can be safely removed")
+	inline char* ParseMacroParameter(eqlib::PlayerClient* pChar, char* szOriginal, size_t BufferSize)
+	{
+		return mq::ParseMacroParameter(szOriginal, BufferSize);
+	}
+
+	template <unsigned int Size>
+	//DEPRECATE("ParseMacroParameter no longer takes a PlayerClient parameter.It can be safely removed")
```

Comment:
I assume these two comments are for future, but what's the reason not to go ahead and deprecate them with this change?